README.md

```diff
@@ -116,6 +118,111 @@ while let Some(batch) = stream.next().await {
 }
 ```
 
+### TLS support
+You can add TLS support to your consumer or producer for secured communication. To enable this, start with specifying the `TlsConnectionOptions`,
+and pass it into an instance of the `ProducerBuilder` or `ConsumerBuilder`.
+
+Example for Producer with TLS support:
+```rust
+let tls_option = TlsConnectionOptions {
+      broker_options: vec!["127.0.0.1:9092".to_string()],
+      key: PathBuf::from("/path_to_key_file"),
+      cert: PathBuf::from("/path_to_cert_file"),
+      cafile: Some(PathBuf::from("/path_to_ca_file")),
+    };
+let topic_name = "my-topic";
+let partition_id = 0;
+
+let message = samsa::prelude::ProduceMessage {
+        topic: topic_name.to_string(),
+        partition_id,
+        key: Some(bytes::Bytes::from_static(b"Tester")),
+        value: Some(bytes::Bytes::from_static(b"Value")),
+        headers: vec![String::from("Key"), bytes::Bytes::from("Value")]
```

Comment:
The `Header` struct is not public to users of the crate :/ we forgot to export it from `lib.rs` so could you please add `pub use crate::protocol::Header;` to the exports in that file?

And then the usage here would be:
```
        headers: vec![samsa::prelude::Header::new(String::from("Key"), bytes::Bytes::from("Value"))],
```

Reply:
Yes I will